source/helpers/bisect.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+
+import features from '../features';
+
+export default async function bisectFeatures(): Promise<Record<string, boolean>> {
+	const bisectedFeatures: string[] | false | undefined = await cache.get('bisect');
+	if (bisectedFeatures === undefined) {
+		return {};
+	}
+
+	if (bisectedFeatures === false) {
+		createMessageBox(<p>Every feature has been disabled. If you still see the change or issue, try disabling the whole extension.</p>, false);
+		return buildOptionsObject([]);
+	}
+
+	if (bisectedFeatures.length === 0) {
+		createMessageBox(<div><p>This process will help you identify what Refined GitHub feature is making changes or causing issues on GitHub. Do you still see the change or issue?</p><p>Note: You can change page here, but don’t use multiple tabs.</p></div>);
+	} else if (bisectedFeatures.length === 1) {
+		createMessageBox(<p>The change or issue is caused by <a href={'https://github.com/sindresorhus/refined-github/blob/main/source/features/' + bisectedFeatures[0] + '.tsx'}><code>{bisectedFeatures[0]}</code></a>.</p>, false);
+	} else {
+		createMessageBox(<p>Can you see the change or issue? ({Math.ceil(Math.log2(bisectedFeatures.length))} steps remaining.)</p>);
+	}
+
+	return buildOptionsObject(bisectedFeatures.slice(0, Math.ceil(bisectedFeatures.length / 2)));
+}
+
+function buildOptionsObject(enabledFeatures: string[]): Record<string, boolean> {
+	return Object.fromEntries(features.list.map(feature => [`feature:${feature}`, enabledFeatures.includes(feature)]));
+}
+
+function createMessageBox(message: Element, yesNoButtons = true): void {
+	document.body.append(
+		<div className="Box p-3" style={{position: 'fixed', bottom: 10, left: '50%', maxWidth: '600px', transform: 'translateX(-50%)'}}>
+			{message}
+			<div className="d-flex flex-justify-between">
+				<div>
+					<button type="button" className="btn" onClick={onEndButtonClick}>Exit</button>
+				</div>
+				{yesNoButtons ?
```

Comment:
I never remember, does this work?
```suggestion
				{yesNoButtons ??
```

Reply:
Doesn't seem like it does.

Reply:
It works, I meant it should replace the ternary operator, not just this line

Reply:
> I meant it should replace the ternary operator

I understood that, but that's not how nullish coalescing works (unless it has another meaning in JSX?)